rs/nns/governance/proto/ic_nns_governance/pb/v1/governance.proto

```diff
@@ -246,6 +246,12 @@ message NeuronInfo {
   optional NeuronType neuron_type = 11;
   // See the Visibility enum.
   optional Visibility visibility = 12;
+  // The last time that following was "refreshed". There are two ways to refresh
+  // the following of a neuron: set following, or vote directly. When this
+  // becomes > 6 months, the amount of voting power that this neuron can
+  // exercise decreases linearly down to 0 over the course of 1 month. After
+  // that, following is cleared, except for ManageNeuron proposals.
+  optional uint64 following_refreshed_timestamp_seconds = 13;
```

Comment:
Should it be documented what happens if a neuron has this field set to None? Or is the idea to make it effectively a non-optional field?